### ISSUE TITLE
Where does mean/std deviation come from?

### ISSUE THREAD

**Issue by carsonswope** [zhangboshen/A2J#41]
For each dataset, there are files called `<dataset>_mean.npy` and `<dataset>_std.npy`. For example, `icvl_mean.npy` and `icvl_std.npy`.

These appear to be the mean and standard deviation of something, and they are used in the preprocessing step of input images. Can anyone explain where these numbers come from and how they would affect using one of the pretrained models in a realtime situation?

**Comment by zhangboshen** [comment]
@carsonswope  Hi,
you can refer to this dissscusion https://github.com/zhangboshen/A2J/issues/29 for more details of mean/std computation. 
And, for the question of how the mean/std affect real time situation, honestly, I dont know, some kind of weird flex might appears with the inappropriate mean/std params, because mean/std is dataset specified, generalization ability of these parms may not be guaranteed.

**Comment by yangsen228** [comment]
@zhangboshen Hello,
I have the same concern about the mean/std value in the real-time situation. If I wanna use your method in a real-time application, what should I do? Can I just retrain the model and remove the mean&std during preprocessing?
Thanks!

**Comment by zhangboshen** [comment]
@yangsen228 You are right, current mean/std parameters are sensitive to cameras. I guess the possible solution is to retrain the model for specific scenarios. And, remove the mean&std during preprocessing can make the regression problem even harder, there might be larger training errors.

**Comment by logic03** [comment]
> @zhangboshen Hello,
> I have the same concern about the mean/std value in the real-time situation. If I wanna use your method in a real-time application, what should I do? Can I just retrain the model and remove the mean&std during preprocessing?
> Thanks!

Have you solve the problem that use this model in you own depth image or video?
